src/clarity-angular/data/stack-view/_stack-view.clarity.scss

```diff
@@ -81,19 +79,19 @@ $clr-stack-font-weight: clr-getTypePropertyValueForDomElement(stackview_text, fo
                 position: relative;
                 width: 0;
                 height: 0;
-                border-top: 0.375rem solid $clr-blue;
+                border-top: 0.375rem solid $clr-stack-bloc-changed-border-top-color;
```

Comment:
Is this named incorrectly? should it be`$clr-stack-block-.....`

Reply:
Correct. Will fix the typo.